modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss

```diff
@@ -67,7 +67,29 @@ $tab-header-height: 38px;
 		line-height: 1.5rem;
 
 		&.active {
-			font-weight: 700;
+			font-weight: 600;
+		}
+	}
+
+	.global-apps-nav-columns {
+		column-count: 1;
+		orphans: 1;
+		widows: 1;
+
+		@include media-breakpoint-up(sm) {
+			column-count: 2;
+		}
+
+		@include media-breakpoint-up(md) {
+			column-count: 3;
+		}
+
+		@include media-breakpoint-up(lg) {
+			column-count: 4;
+		}
+
+		.col-md {
+			display: inline-block;
```

Comment:
This is pretty cool, I figured I could achieve the same result with `col-lg-3 col-sm-6 col-md-4` but I realized that the layout format is distributed differently.